apps/DesktopStreamer/CMakeLists.txt

```diff
@@ -38,8 +38,8 @@ if(APPLE)
   else()
     list(APPEND DESKTOPSTREAMER_LINK_LIBRARIES "-framework CoreGraphics")
   endif()
-elseif(WIN32)
-  list(APPEND DESKTOPSTREAMER_LINK_LIBRARIES Ws2_32)
+elseif(MSVC)
+  list(APPEND DESKTOPSTREAMER_LINK_LIBRARIES Ws2_32 Qt5::WinMain)
```

Comment:
I'm trying to see if this couldn't be done by common_gui_application() or common_qt_support(). From what I get it is only needed for GUI applications.